client/src/components/workspaces/WorkspaceCard.vue

```diff
@@ -3,11 +3,11 @@
 <template>
   <div
     class="card"
-    @click="$emit('click', $event, workspace)"
+    @click="$emit('click', workspace)"
```

Comment:
The `$event` should still be present here.

Don't see the use of this component only in this particular context. This component should be usable in different ways. What if we want to open a context menu on click? We need the event to know where to position it.

All the emits should take an optional `Event` parameter, and if we can provide it, we do.

This component provides an API, it shouldn't be strongly coupled with the use we make of it. Instead, it should be as generic as possible.

This comment stands for other similar emits.